bin/user/interceptor.py

```diff
@@ -2361,9 +2374,13 @@ class Parser(Consumer.Parser):
             'wh25batt': 'wh25_battery',
             'wh26batt': 'wh26_battery',
             'wh40batt': 'wh40_battery',
+            'wh57batt': 'wh57_battery',
             'wh65batt': 'wh65_battery',
             'pm25_ch1': 'pm2_5',
             'pm25batt1': 'pm25_battery',
+            'lightning_num': 'lightning_strikes_total',
```

Comment:
Actually, I'll probably rename this to `day_lightning_strikes` or something similar, since it wraps at midnight.